pageserver/src/http/routes.rs

```diff
@@ -160,10 +160,9 @@ async fn timeline_create_handler(mut request: Request<Body>) -> Result<Response<
     let request_data: TimelineCreateRequest = json_request(&mut request).await?;
     check_permission(&request, Some(tenant_id))?;
 
+    let tenant = tenant_mgr::get_tenant(tenant_id, true)?;
```

Comment:
Just a heads up: if #2461 is merged before this, the error will need to be explicitly converted with something like `map_err(ApiError::BadRequest)`

Reply:
Turns out, `timeline_create_handler` returns `anyhow::Result`, so no extra conversion is needed.
Yet I dislike that standard prelude `std::core::Result` gets shadowed by `anyhow::Result`, I think we can adjust it bit by bit when touching the related code.

Reply:
I'm not sure what you mean - currently `timeline_create_handler` has a return type of `Result<Response<Body>, ApiError>`.

If you're referring to how `anyhow::Result` is imported at the top of the file, so that `Result` is referencing `anyhow`'s result type, rather than the standard library's, note that the definition of [`anyhow::Result`](https://docs.rs/anyhow/latest/anyhow/type.Result.html) is:
```rust
type Result<T, E = anyhow::Error> = std::resut::Result<T, E>;
```
so when we explicitly provide the error type, it _is_ the standard library's `Result` type

Reply:
I've meant that 

> the error will need to be explicitly converted

event actually did not happen, so wanted to warn about this, and just noted, that I like `anyhow::Result` or regular `Result`, but not the imported version that gets aliased to something.

Either way, the code seems to compile (as CI should show eventually) and I've not touched the method signatures, so everything seems to be fine 🙂

Reply:
Ah ok - still not entirely sure what you mean, but maybe it's not super crucial :)

>  I like anyhow::Result or regular Result, but not the imported version that gets aliased to something

I feel the same way. In general, I tend to prefer the `module::Result` syntax in return types (`io::Result` and `fmt::Result`) as well